### ISSUE TITLE
A11y_VSCode_Output panel_ScreenReader:Screen reader does not annunce anything when click on the toggle scroll lock

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#76696]
**Environment Details:**
VSCode Version : 1.36.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.3.1

**Repro Steps:**
1) Open Visual studio code editor  
2) Press ctrl+shift+u
3) Navigate to the toggle scroll lock button using keyboard
4) Observe the Screen reader announcement. 

**Actual:**
Screen reader does not annoucne anything  when click on the toggle scroll lock button.

**Expected:**
Screen reader should annoucne the state of the scroll lock button.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader will not able to understand whether state of the scroll lock button is lock or unlocked 

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:** 
[A11y_VSCode_Output panel_ScreenReader_NoAnnouncementOfStateOfButton.pptx](https://github.com/microsoft/vscode/files/3362146/A11y_VSCode_Output.panel_ScreenReader_NoAnnouncementOfStateOfButton.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Thanks for reporting this.
I have changed the hovere on the output scroll lock to change based on state. This way the user will now get different feedback if the output scrolling is turned on or off.
You can see the changes in the tomorrow's vscode insiders or next vscode stable
